app/src/main/java/org/wikipedia/page/shareafact/ShareHandler.java

```diff
@@ -250,10 +250,10 @@ public boolean onMenuItemClick(MenuItem item) {
             rootView.findViewById(R.id.share_as_image_button)
                     .setOnClickListener((v) -> {
                         String introText = context.getString(R.string.snippet_share_intro,
-                                title.getDisplayText(),
+                                StringUtil.fromHtml(title.getDisplayText()),
                                 UriUtil.getUrlWithProvenance(context, title, R.string.prov_share_image));
-                        ShareUtil.shareImage(context, resultBitmap, title.getDisplayText(),
-                                title.getDisplayText(), introText);
+                        ShareUtil.shareImage(context, resultBitmap, title.getText(),
```

Comment:
Using `getText()` instead of `getDisplayText()` fixes the "interpreted" issue shown in the bug report.